src/Core/Constants.cs

```diff
@@ -124,6 +124,7 @@ public static class Constants
         public static string PushCurrentTokenKey(string userId) => $"pushCurrentToken_{userId}";
         public static string ShouldConnectToWatchKey(string userId) => $"shouldConnectToWatch_{userId}";
         public static string ScreenCaptureAllowedKey(string userId) => $"screenCaptureAllowed_{userId}";
+        public static string PendingAdminAuthRequest(string userId) => $"pendingAdminAuthRequest{userId}";
```

Comment:
⛏️ In order to follow the pattern an underscore must be placed before the user id, or is there a reason not to use it here?
```suggestion
        public static string PendingAdminAuthRequest(string userId) => $"pendingAdminAuthRequest_{userId}";
```